pkg/cloud/azure/provider.go

```diff
@@ -1099,8 +1100,10 @@ func (az *Azure) NodePricing(key models.Key) (*models.Node, models.PricingMetada
 
 	meta := models.PricingMetadata{}
 
+	azurePricingExists := true
 	if az.Pricing == nil {
-		return nil, meta, fmt.Errorf("Unable to download Azure pricing data")
+		azurePricingExists = false
+		log.DedupedWarningf(5, "Azure pricing data not found")
```

Comment:
This log maybe belongs somewhere else

Reply:
It should be reflected in a log when set initially anyhow, I've just removed it.